game/CoreServices/Integrity.rpy

```diff
@@ -1,8 +1,13 @@
 ## Integrity.rpy
 # System Integrity Protection via GOBFADU policy
-# Author: Monika/D. Va (@GanstaKingofSA)
+# Author: Monika/D. Va (@GanstaKingofSA), Marquis Kurt (@alicerunsonfedora)
 # Copyright: (C) 2018
+image integrity_message = Text("Checking system integrity...", font="Resources/systemfont/Light.ttf", size=14, style="_default")
 
 label Integrity:
-    call gobfadupolicygobfadu
-return
\ No newline at end of file
+    # Show Integrity check message before loading
+    show integrity_message zorder 3:
+        xalign 0.5
+        yalign 0.8
+    call gobfadu_policy_gobfadu
```

Comment:
has this been tested? because if you run `somethingcall` and straight up just changed it from definition level it'll be undefined

Reply:
When I start up the game, it works fine. But, there are some odd "perks" about it...
- I don't know why this happens, but the GOBFADU policy loader always shows a black screen first and then the AliceOS boot logo (weird)
- If the RPYCs are not built before the game runs, it will get stuck at "Checking system integrity..." (so I had to fix the `run.py` file so that it compiled them before running it.
- I haven't tried running the policy by itself. All I have done is changed the names so it follows snake case.